src/App.tsx

```diff
@@ -1,24 +1,14 @@
-import { useEffect, useState } from "react";
 import "./globals.css";
 import styles from "./App.module.css";
+import { Outlet } from "react-router";
+import NavBar from "./components/NavBar/NavBar";
+// import Loading from "./pages/Loading/Loading";
 
 function App() {
-  const [message, setMessage] = useState("");
-
-  useEffect(() => {
-    async function doFetch() {
-      const response = await fetch("/api/hello");
-      const result = await response.json();
-      setMessage(result.message);
-    }
-    doFetch();
-  }, []);
-
   return (
     <div className={styles.app}>
-      <header className={styles["app-header"]}>
-        <p className={styles.text}>Welcome to your new project 💥 {message}</p>
```

Comment:
If I see this corretly, you should remove `{message}` since it is no variable anymore

Reply:
It's deleted :)